### PR TITLE
Fix e2e tests after changing manual verification UI

### PR DESCRIPTION
**Pull request by fzhao99**
## Related Issue or Background Info
Fixed e2e test to account for new account verification flow made via #2943 by changing the selectors around. Also added in some extra unit tests for the PendingOrganizations component.

Got trigger happy and merged change from #3092 before fixing the e2e test. This PR fixes that!

Diff only displays changes after commit 330a8d8, everything else was ok'ed in the other PR.

### Testing
- [ ] Double check that new tests look good (should just be minor tweaks to Cypress selectors.)

### CODE REVIEW

File: frontend/cypress/integration/01-organization_sign_up_spec.js [hunk 1]
```diff
@@ -33,12 +33,12 @@ describe("Organization sign up", () => {
...
-    cy.contains("Organizations Pending Identity Verification");
+    cy.get("[data-cy=pending-orgs-title]").should("be.visible");
   });
   it("verifies the org", () => {
-    cy.get('input[name="identity_verified"]+label').first().click();
-    cy.contains("Save Changes").first().click();
-    cy.contains("Identity verified for 1 organization", { timeout: 30000 });
+    cy.get(".sr-active-button").first().click();
+    cy.get("#verify-button").click();
+    cy.contains("Identity verified for Beach Camp", { timeout: 30000 });
```
Comment (nickclyde) [inline frontend/cypress/integration/01-organization_sign_up_spec.js:41]: Cool, thanks for updating :+1:

File: package.json [hunk 2]
```diff
@@ -20,5 +20,8 @@
     "e2e": "cd frontend; yarn e2e; cd ..",
     "spotless": "cd backend; ./gradlew spotlessApply; cd ..",
     "db": "docker-compose -f backend/docker-compose.yml exec db psql -U postgres simple_report"
+  },
+  "dependencies": {
+    "eslint-plugin-cypress": "^2.12.1"
```
Comment (nickclyde) [inline package.json:25]: I don't think this needs to be added to the `package.json` in the root of the repo, it's already in `frontend/package.json`

**Review by nickclyde — CHANGES_REQUESTED** [review]

File: yarn.lock [hunk 3]
```diff
@@ -233,6 +233,13 @@ escape-string-regexp@^1.0.5:
   resolved "https://registry.yarnpkg.com/escape-string-regexp/-/escape-string-regexp-1.0.5.tgz#1b61c0562190a8dff6ae3bb2cf0200ca130b86d4"
   integrity sha1-G2HAViGQqN/2rjuyzwIAyhMLhtQ=
 
+eslint-plugin-cypress@^2.12.1:
```
Comment (nickclyde) [inline yarn.lock:236]: Sorry, can you revert the changes to `yarn.lock` as well?

**Review by nickclyde — APPROVED** [review]
:+1:

**Review by nathancrtr — APPROVED** [review]

**Pull request merged**